### ISSUE TITLE
doesn't display properly non-ascii username

### ISSUE THREAD

**Issue by glensc** [inconshreveable/ngrok#481]
i've connected with my github handle.

you see there's extra space after `ä`:
```
Session Status                online
Account                       Elan Ruusamä e (Plan: Free)
Version                       2.2.8
Region                        Europe (eu)
```

![image](https://user-images.githubusercontent.com/199095/36790778-94da4b28-1c9e-11e8-8b0d-24e225f4bc28.png)

**Comment by russorat** [comment]
Thank you for opening this issue! As of April 2016, the ngrok service has permanently moved to https://ngrok.com/. To join the community and file bug reports or feature enhancements for the ngrok agent, please go to https://github.com/ngrok/ngrok.

This repository is no longer actively maintained and will be archived soon to reduce confusion with the latest ngrok versions. We thank you for the continued support of ngrok and look forward to seeing you over at [ngrok.com](https://ngrok.com/) and the [ngrok Community Repo](https://github.com/ngrok/ngrok).

**Comment by russorat** [comment]
This is fixed in the latest ngrok agent v3.10
